x-pack/plugins/lens/public/indexpattern_datasource/field_item.tsx

```diff
@@ -552,3 +629,38 @@ function FieldItemPopoverContents(props: State & FieldItemProps) {
   }
   return <></>;
 }
+
+const DragToWorkspaceButton = ({
+  field,
+  dropOntoWorkspace,
+  isEnabled,
+}: {
+  field: Dragging;
+  dropOntoWorkspace: DatasourceDataPanelProps['dropOntoWorkspace'];
+  isEnabled: boolean;
+}) => {
+  const buttonTitle = isEnabled
+    ? i18n.translate('xpack.lens.indexPattern.moveToWorkspace', {
+        defaultMessage: 'Add field to workspace',
```

Comment:
Any chance we can get the field name into this? So it reads: `Add ${fieldName} to workspace`